src/configureStore.spec.tsx

```diff
@@ -0,0 +1,9 @@
+import configureStore from './configureStore';
+
+describe(__filename, () => {
+  it('registers all the reducers', () => {
+    const store = configureStore();
+
+    expect(Object.keys(store.getState())).toEqual(['api', 'example']);
```

Comment:
Nit: It seems like this might be a hassle to maintain, but I guess it's worth having.